src/tests/functional/plugin/shared/src/low_precision_transformations/concat_transformation.cpp

```diff
@@ -34,15 +36,27 @@ std::string ConcatTransformation::getTestCaseName(const testing::TestParamInfo<C
     return result.str();
 }
 
-InferenceEngine::Blob::Ptr ConcatTransformation::GenerateInput(const InferenceEngine::InputInfo &info) const {
-    ngraph::PartialShape inputShape;
-    ngraph::element::Type netPrecision;
-    std::string targetDevice;
-    ConcatTransformationTestValues testValues;
-    std::tie(netPrecision, inputShape, targetDevice, testValues) = this->GetParam();
+ov::test::utils::InputsMap ConcatTransformation::get_input_map() {
+    auto generate_default = [](const std::shared_ptr<ngraph::Node>&node,
```

Comment:
```suggestion
    auto generate_default = [](const std::shared_ptr<ov::Node>&node,
```
Please grep for `ngraph` keyword in order to remove legacy namespace at all